composer.json

```diff
@@ -0,0 +1,36 @@
+{
+    "name": "ubmannheim/publist4ubma2",
```

Comment:
We could choose here also another namespace `ub-mannheim` or maybe `unima`. What do you think? We should then just make sure we can get a hold in in packagist.